app/components/tracks/TracksController.js

```diff
@@ -31,6 +33,65 @@ tracksModule.controller('TracksController',
                 }, function (error) {
                     console.error(error);
                 });
+
+            ApiJsonFactory.getJson('sessions')
+                .then(function (response) {
+                    tc.Sessions = response.data.sessions;
+                    $sessionStorage.sessions = tc.Sessions;
+                }, function (error) {
+                    console.error(error);
+                });
         }
 
-}]);
\ No newline at end of file
+        tc.showSession = function(track, event) {
```

Comment:
I think this should help: http://stackoverflow.com/questions/12008908/angularjs-how-can-i-pass-variables-between-controllers